app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable shared among all instances of `Animal`. This is correct if you want to track all alive animals globally. However, ensure this is the intended behavior, as it might lead to unexpected results if not managed carefully.